x/nft/keeper/querier.go

```diff
@@ -101,7 +106,11 @@ func queryDenom(ctx sdk.Context, req abci.RequestQuery, k Keeper, legacyQuerierC
 		return nil, sdkerrors.Wrap(sdkerrors.ErrJSONUnmarshal, err.Error())
 	}
 
-	denom, _ := k.GetDenom(ctx, params.ID)
+	denom, err := k.GetDenom(ctx, params.ID)
+
+	if err != nil {
+		return nil, err
```

Comment:
should this do some `sdkerrors.Wrap(sdkerrors.<some error, e.g. invalid request>, err.Error())`?

Reply:
`GetDenom` already returns wrapped error